code/workspaces/web-app/src/components/settings/AddUserPermissions.js

```diff
@@ -134,7 +134,6 @@ export function AddUserButton({ currentUserId, currentUserSystemAdmin, selectedU
       <div>
         <PrimaryActionButton
           className={classes.addButton}
```

Comment:
Just doing some testing and it behaves strangely if you don't enter anything and just hit submit (some slowness/freezing). I wonder if we could still disable if it's empty?

Reply:
I can confirm I was getting the same freezing behaviour when an empty value was submitted.  The button is now disabled when there is a disabledMessage.  I think I accidentally removed that in an earlier commit :( , but it didn't show up in tests, so well spotted 🥇